tests/test_panel_segmentation.py

```diff
@@ -18,7 +18,6 @@ def assert_isinstance(obj, klass):
     assert isinstance(obj, klass), f'got {type(obj)}, expected {klass}'
 
 
-@pytest.fixture()
```

Comment:
Not sure what all these fixture decorators were for -- are they actually needed here?  It's not a usage of fixtures that I'm familiar with and the test suite seems to run as intended without them